File: ts/examples/serve_config.ts
import { Client } from "../src/index.js";
import { jsonReplacer } from "../src/json.js";

const client = new Client();
const config = await client.getServeConfig();
console.error(`ETag: ${config.ETag}`);
console.log(JSON.stringify(config, jsonReplacer, 2));
client.destroy();

File: rust/examples/serve_config.rs
use tslocalapi::Client;

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let client = Client::new();
    let config = client.get_serve_config().await?;
    eprintln!("ETag: {}", config.e_tag);
    println!("{}", serde_json::to_string_pretty(&config)?);
    Ok(())
}
